medusa/search/core.py

```diff
@@ -253,7 +245,7 @@ def pick_best_result(results, show):  # pylint: disable=too-many-branches
     """
     results = results if isinstance(results, list) else [results]
 
-    logger.log(u'Picking the best result out of ' + str([x.name for x in results]), logger.DEBUG)
+    log.debug(u'Picking the best result out of {0}', [x.name for x in results])
```

Comment:
Yes it will print the repr as normal.  Only a `collections.Mapping` descendant (like `dict`) as a single param require special treatment because it will try to unpack them first as kwargs.